### ISSUE TITLE
Localhost doesn't work as expected

### ISSUE THREAD

**Issue by Naggertooth** [fido-alliance/webauthn-demo#20]
+1

Firefox 71.0 + node v12.13.1 + Linux Mint 19.2

Developers, your demo site works for me fine but this demo from localhost:3000 - nothing happens. I see login form, i do register -> fill form -> submit -> i see login page again -> submit name -> i see login page and so on.

In development tools in Net i see "304" GET responses for each request only... Everywhere...

_Originally posted by @Perlover in https://github.com/fido-alliance/webauthn-demo/issues/19#issuecomment-565381406_

**Comment by Naggertooth** [comment]
That's because you need to run it on https
I m using heroku for fast tests

**Comment by Perlover** [comment]
https://github.com/fido-alliance/webauthn-demo/issues/21

**Comment by forevermatt** [comment]
If you take the `static/js/webauthn.auth.js` and `routes/webauthn.js` files from the "completed-demo" branch and copy/paste them into the latest "master" branch's version of the files, the application then can work.

I used traefik to get https for local development work, using a Docker container running traefik and a separate Docker container running this webauthn-demo code (using the `node:6` Docker image). Once I also updated the webauth-demo `config.json` file's "origin" to match the domain name I have aimed at my traefik container (with protocol, without trailing slash after domain name), the webauthn-demo works.
